src/templater.rb

```diff
@@ -19,16 +19,16 @@
 # For more information on the Alces Metalware, please visit:
 # https://github.com/alces-software/metalware
 #==============================================================================
+
 require "erb"
-require "ostruct"
-require "yaml"
 
 require "constants"
 require 'metal_log'
 require 'exceptions'
 require 'iterable_recursive_open_struct'
 require 'magic_namespace'
 
+
```

Comment:
Not really, I just like to leave a couple of spaces between my `require`s and the start of a class definition! Probably a habit picked up from [Python](https://www.python.org/dev/peps/pep-0008/#blank-lines).